### PR TITLE
Migrate Kafka components installation to new components

### PR DESCRIPTION
**Pull request by pierDipi**
Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

-
-
-

### CODE REVIEW

File: docs/install/yaml-install/eventing/install-eventing-with-yaml.md [hunk 1]
```diff
@@ -50,8 +50,28 @@ Follow the procedure for the Channel of your choice:
...
-    1. Install [Strimzi](https://strimzi.io/quickstarts/).
-    1. Install the Apache Kafka Channel for Knative from the [`knative-sandbox` repository](https://github.com/knative-sandbox/eventing-kafka).
+
+    The following commands install the KafkaChannel and run event routing in a system
+    namespace. The `knative-eventing` namespace is used by default.
+
+    1. Install the Kafka controller by running the following command:
+
+        ```bash
+        kubectl apply -f {{ artifact(org="knative-sandbox",repo="eventing-kafka-broker",file="eventing-kafka-controller.yaml")}}
```
Comment (matzew) [inline docs/install/yaml-install/eventing/install-eventing-with-yaml.md:60]: do we need to mention how to disable the other controllers (e.g. for the source)?

File: docs/install/yaml-install/eventing/install-eventing-with-yaml.md [hunk 2]
```diff
@@ -265,10 +292,16 @@ The following tabs expand to show instructions for installing each Eventing exte
 
 === "Apache Kafka Source"
 
-    * Install the Apache Kafka Source by running the command:
+    1. Install the Apache Kafka Source by running the command:
```
Comment (matzew) [inline docs/install/yaml-install/eventing/install-eventing-with-yaml.md:295]: similar q regards to the `--disable-controller` argument

File: docs/eventing/channels/channels-crds.md [hunk 3]
```diff
@@ -16,10 +16,9 @@ This is a non-exhaustive list of the available Channels for Knative Eventing.
...
 [GCP PubSub](https://github.com/google/knative-gcp) | Proof of Concept | None | Channels are backed by [GCP PubSub](https://cloud.google.com/pubsub/).
 [InMemoryChannel](https://github.com/knative/eventing/tree/{{ branch }}/config/channels/in-memory-channel/README.md) | Proof of Concept | None | In-memory channels are a best effort Channel. They should NOT be used in Production. They are useful for development.
-[KafkaChannel - Consolidated](https://github.com/knative-sandbox/eventing-kafka/tree/{{ branch }}/pkg/channel/consolidated/README.md) | Proof of Concept | None | Channels are backed by [Apache Kafka](http://kafka.apache.org/) topics. The original Knative KafkaChannel implementation which utilizes a single combined Kafka Producer / Consumer deployment.
-[KafkaChannel - Distributed](https://github.com/knative-sandbox/eventing-kafka/tree/{{ branch }}/pkg/channel/distributed/README.md) | Proof of Concept | None | Channels are backed by [Apache Kafka](http://kafka.apache.org/) topics. An alternate KafkaChannel implementation, contributed by SAP's [Kyma](https://kyma-project.io/) project, which provides a more granular deployment of Producers / Consumers.
```
Comment (aliok) [inline docs/eventing/channels/channels-crds.md:24]: I wouldn't delete these directly. Instead, let's mark them deprecated.

File: docs/eventing/channels/channels-crds.md [hunk 4]
```diff
@@ -16,10 +16,9 @@ This is a non-exhaustive list of the available Channels for Knative Eventing.
...
+[KafkaChannel](https://github.com/knative-sandbox/eventing-kafka-broker/tree/{{ branch }}/README.md) | Beta             | None | Channels are backed by [Apache Kafka](http://kafka.apache.org/) topics.
```
Comment (aliok) [inline docs/eventing/channels/channels-crds.md:23]: I created https://github.com/knative/docs/pull/5001 for this line. Linking to https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/docs/channel/README.md

**Comment by aliok** [comment]
LGTM

**Comment by abrennan89** [comment]
/approve

Merging since this was LGTM'd by @aliok

**Pull request merged**